modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.5.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.5 that need your attention.
+You can also read {owncloud-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+- To improve the performance of addressbook search queries (e.g., when looking for federated users to share with), a 
+https://github.com/owncloud/core/pull/37152[migration step] adds indices for the columns `addressbookid`, `name` and `value` on the `oc_cards_properties` table. The impact on upgrade duration can be high depending on the number of rows of the mentioned columns.
+- To prepare for the new background job for change detection in federated shares (xref:release_notes.html#new-background-job-for-change-detection-in-federated-shares[see below]), a https://github.com/owncloud/core/pull/37391[migration step] adds a new column (`lastscan`) to the `oc_share_external` table. The impact on upgrade duration depends on the number of rows in `oc_share_external`.
+- To enable storing complex WebDAV properties, a https://github.com/owncloud/core/pull/37314[migration step] adds a new column (`propertytype`) to the `oc_properties` and `oc_dav_properties` tables. The impact on upgrade duration depends on the number of rows in `oc_properties` and `oc_dav_properties`.
+- To facilitate the transition to the new licensing mechanism (xref:release_notes.html#changes-in-delivery[see below]) a https://github.com/owncloud/core/pull/37512[migration step] will disable the `enterprise_key` app during the upgrade. This step does not have an impact on upgrade duration.
+
+=== PHP 7.1 Support Discontinued
+
+As xref:release_notes.html#php-71-deprecation-note[announced], in the previous minor release of ownCloud Server, from version 10.5 onward, ownCloud Server **no longer supports PHP 7.1**. 
+If you're running on PHP 7.1 or below, it is necessary to upgrade PHP **prior** to conducting the upgrade to Server 10.5. 
+See the xref:installation/system_requirements.adoc[system requirements] for more information.
+
+NOTE: If you're using the official Docker containers or the Univention appliance, this has been taken care of already.
+
+=== Official PHP 7.4 Support
+
+ownCloud Server 10.5 officially supports PHP 7.4. 
+The Server Core and all apps maintained by ownCloud have received a full QA cycle and are proven to work reliably with PHP 7.4.
+If you are still running a PHP version < 7.2, you must upgrade PHP before upgrading ownCloud Server as lower versions are not supported anymore.
+
+Summarizing, ownCloud Server 10.5 supports the PHP versions **7.2, 7.3 and 7.4**.
+
+TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud Documentation] for more information.
```

Comment:
`supports the PHP versions **7.2, 7.3 and 7.4**`
Following the link tells to use php7.3
It may be good to include that ownCloud suggests using php7.3 onwards, to be inline with the doc link 
![image](https://user-images.githubusercontent.com/3321281/88456576-d56b4680-ce7e-11ea-965b-d01084a560f7.png)

Reply:
Yes, that is the recommendation. A little below are the supported versions. We need to change `{supported-php-versions}` (can't find it), remove `7.1` and add `7.4`.

Reply:
Proposing:
` for more information.` --> ` for the recommended PHP version and for more information.`